projects/demo/src/modules/components/arc-chart/examples/1/index.html

```diff
@@ -1,11 +1,12 @@
-<tui-input-number
-    decimal="never"
-    class="index-controller"
-    [step]="1"
-    [(ngModel)]="activeItemIndex"
->
-    activeItemIndex
-</tui-input-number>
+<tui-textfield class="index-controller">
```

Comment:
Approve but please fix the example on failed screenies

Reply:
Temporary revert changes inside `ArcChart` because of this bug:
* https://github.com/taiga-family/taiga-ui/issues/10202

This example will be refactored later